### ISSUE TITLE
KP.3.3 with S:S60F and S:L828V (C>G) (3 , Japan)

### ISSUE THREAD

**Issue by FedeGueli** [sars-cov-2-variants/lineage-proposals#2065]
Three recent KP.3.3  samples from japan features a very rare C>G mutation S:L828V.
S:L828 in the past has been thought to be a key to shift from Respiratory tract to GIT tract preference ,
i hope @ryhisner or @thomasppeacock could elaborate about this. No idea about V what could do there .
Interestingly it features also S:S60F that was rare before Jn.1 while present in more than 250 samples since JN.1 emerged.
It has also G29779T  in the 3' UTR
and Orf1a:A2199T that is NSP3_A1381T that was in some bAt viruses in Cambodia

KP.3.3 >> C5221T > C21219T > C12073T, A14421G, C21721T > A6739G, Orf1a:A2199T (G6860A), C21741T, S:L828V (C24044G), C27513T, G29779T 
Query:C21741T, C24044G
Samples: 3 Japan (Yamaguchi)
Tree:
<img width="1178" alt="Screenshot 2024-09-20 alle 10 46 02" src="https://github.com/user-attachments/assets/d5a7dd60-6e3a-4d74-9ba0-f7d472d610c9">
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_585dd_d32830.json?c=userOrOld&label=id:node_7068293

**Comment by Over-There-Is** [comment]
1 with S:V143-, S:Y144F

**Comment by aviczhl2** [comment]
> 1 with S:V143-, S:Y144F

Shall be S:V143F?

**Comment by Over-There-Is** [comment]
> > 1 with S:V143-, S:Y144F
> 
> Shall be S:V143F?

I think Fede won't agree.

**Comment by FedeGueli** [comment]
> > > 1 with S:V143-, S:Y144F
> > 
> > 
> > Shall be S:V143F?
> 
> I think Fede won't agree.

I agree. @Over-There-Is there is a difference between anticipating the mutation after the deletion before it and instead correcting when the pipeline reads it bad, Deletions are very hard to be reverted especially with a same nuc sequence, so in this case it is clearly S:V143F. Different discourse for N:S33C that is AFTER the deletion but you can read either as N:G30C because you have three missing AA: in that case the pipeline reads the deletion correctly but you are over-clever in reading it as the 2AA are missing. In fact if you anticipate the mutation to the last AA before the deletion then you should go with the same thing until the end of the protein. Is it clearer now?

**Comment by FedeGueli** [comment]
> > > > 1 with S:V143-, S:Y144F
> > > 
> > > 
> > > Shall be S:V143F?
> > 
> > 
> > I think Fede won't agree.
> 
> I agree. @Over-There-Is there is a difference between anticipating the mutation after the deletion before it and instead correcting when the pipeline reads it bad, Deletions are very hard to be reverted especially with a same nuc sequence, so in this case it is clearly S:V143F. Different discourse for N:S33C that is AFTER the deletion but you can read either as N:G30C because you have three missing AA: in that case the pipeline reads the deletion correctly but you are over-clever in reading it as the 2AA are missing. In fact if you anticipate the mutation to the last AA before the deletion then you should go with the same thing until the end of the protein. Is it clearer now?

So the rule could be the following if the pipeline keeps the deletion in the original position and the read is correect keep it that way as the pipleines read it. if the pipelines inverts things as this you revert them keeping the deletion in the same residue. I hope to have been clear enough

**Comment by FedeGueli** [comment]
And btw if you write N:G30C you should write N:G30C N:31-33del, if not explicited it is wrong.
